apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -175,6 +181,55 @@ func validateDNSName(hostaddress string) error {
 	return nil
 }
 
+func validateAnnotations(host *BareMetalHost) []error {
+	var errs []error
+	var err error
+
+	for annotation, value := range host.Annotations {
+
+		switch {
+		case annotation == StatusAnnotation:
+			err = validStatusAnnotation(value)
```

Comment:
nit: all these new functions should probably start with "validate", not "valid"

Reply:
fixed it